examples_tests/13.BRDF_Explorer/BRDFExplorerApp.cpp

```diff
@@ -374,19 +388,33 @@ R"(
 		// there are better identities to get all of these
 		const vec3 V = normalize(uEyePos - WorldPos);
 		const vec3 L = normalize(relLightPos);
```

Comment:
optimize this, you already calculate the inverseqsrt of `relLightPosLen2` above!